builtin/common/vector.lua

```diff
@@ -1,6 +1,18 @@
 
 vector = {}
 
+function vector.from(x, y, z)
```

Comment:
This should be the new `new()` and should check arguments are not `nil`.

Reply:
Done.